Code2CollegeBankingProject/BankingUI/views.py

```diff
@@ -109,26 +107,32 @@ def forgotpassword(request):
         else:
             userinfo = {"response": "This username doesn't exist"}
             return render(request, "forgotpassword.html", userinfo)
-    return render(request, "forgotpassword.html")
+    return render(request,"forgotpassword.html")
 
 
 @login_required(redirect_field_name="index")
 def account(request):
     customer_transactions = Transactions.objects.filter(account=request.user.id)
     customer_amount = 0
-    for transactions in customer_transactions:
-        if transactions.transactiontype == "Deposit":
-            customer_amount += transactions.amount
-        if transactions.transactiontype == "Withdraw":
-            customer_amount -= transactions.amount
+    balance = []
+    i = 0
+    for transaction in customer_transactions:
+        if transaction.transactiontype == "Deposit":
+            customer_amount += transaction.amount
+        if transaction.transactiontype == "Withdraw":
+            customer_amount -= transaction.amount
+        balance.append(customer_amount)
+        i += 1
```

Comment:
What is `i` doing here? This variable isn't being used in the view correct?

Reply:
it was a attempt to solve the problem haha I removed it